go/tasks/plugins/array/k8s/launcher.go

```diff
@@ -52,6 +52,14 @@ func ApplyPodPolicies(_ context.Context, cfg *Config, pod *corev1.Pod) *corev1.P
 	return pod
 }
 
+func applyNodeSelectorLabels(_ context.Context, cfg *Config, pod *corev1.Pod) *corev1.Pod {
```

Comment:
should function start with uppercase? : `ApplyNodeSelectorLabels`

Reply:
Since I'm not exporting this function, it should be fine lowercase.

Reply:
ah, gotcha! 👍